src/wp-includes/class-walker-nav-menu.php

```diff
@@ -286,4 +317,24 @@ public function end_el( &$output, $data_object, $depth = 0, $args = null ) {
 		$output .= "</li>{$n}";
 	}
 
+	/**
+	 * Builds a string of HTML attributes from an array of key/value pairs.
+	 * Empty values are ignored.
+	 *
+	 * @since 6.2.0
+	 *
+	 * @param  array $atts Optional. An array of HTML attribute key/value pairs. Default empty array.
+	 * @return string A string of HTML attributes.
+	 */
+	protected function build_atts( $atts = array() ) {
+		$attributes = '';
```

Comment:
Having `$atts`, `$attributes` and `$attr` all within one function is a little unclear, let's change the outcome to something clearer. (I know this was an existing issue but it's good to improve while moving the code.)

```suggestion
		$attribute_string = '';
```

Reply:
By the same token I'd consider renaming the method to `build_attributes()` in the context of "you don't build bricks/$atts, you build houses/$attributes". Besides, `build_attributes` requires no expansion, so it's cleaner.